test/new-e2e/containers/eks_test.go

```diff
@@ -38,13 +38,13 @@ func TestEKSSuite(t *testing.T) {
 		"ddtestworkload:deploy": auto.ConfigValue{Value: "true"},
 	}
 
-	_, stackOutput, err := infra.GetStackManager().GetStack(ctx, "eks-cluster", stackConfig, eks.Run, false)
-	require.NoError(t, err)
-
 	t.Cleanup(func() {
 		infra.GetStackManager().DeleteStack(ctx, "eks-cluster")
 	})
 
+	_, stackOutput, err := infra.GetStackManager().GetStack(ctx, "eks-cluster", stackConfig, eks.Run, false)
```

Comment:
As [stack.Up](https://github.com/DataDog/datadog-agent/blob/01e56eddd0cd5c44a54e1f602e7f239306c4feeb/test/new-e2e/utils/infra/stack_manager.go#L111) is the last action of `GetStack`, if `GetStack` returns en error, then the stack shouldn't be created. Did I miss something?